frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/InsideHearingMainPage.js

```diff
@@ -163,7 +163,7 @@ const InsideHearingMainPage = () => {
     } else {
       setTranscriptText(newText);
 
-      if (Object.keys(hearing).length === 0) {
+      if (Object.keys(hearing)?.length === 0) {
```

Comment:
if hearing is an object Object.keys will always return an array, why are we conditionally chaining on length ?

Reply:
addressed